test/integration-tests.js

```diff
@@ -0,0 +1,26 @@
+// Collect all integration tests here.
+//
+// To run, insert the following code into a telerivet service
+//   ```
+//   var test = require('ext/roster/test/integrations/tests');
+//   test();
```

Comment:
@gregstuderoaf this should be all that's needed to run the tests now in a telerivet service

Reply:
Awesome.  README.md in the test folder would be cool, but not req'd for this patch.

We should also add a caveat that this code won't be able to test certain kinds of conversational logic flows, but again awesome as-is.